client/chainlink.go

```diff
@@ -62,22 +64,43 @@ func NewChainlink(c *ChainlinkConfig, httpClient *http.Client) (Chainlink, error
 	return cl, cl.SetSessionCookie()
 }
 
+func FundTemplateNodes(blockchainClient BlockchainClient, wallets BlockchainWallets, nodes []Chainlink, ethAmount int64, linkAmount int64) error {
+	for _, node := range nodes {
+		nodeEthKeys, err := node.ReadETHKeys()
+		if err != nil {
+			return err
+		}
+		primaryEthKey := nodeEthKeys.Data[0]
+
+		err = blockchainClient.Fund(
+			wallets.Default(),
+			primaryEthKey.Attributes.Address,
+			big.NewInt(ethAmount), big.NewInt(linkAmount),
+		)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 // ConnectToTemplateNodes assumes that 5 template nodes are running locally, check out a quick setup for that here:
 // https://github.com/smartcontractkit/chainlink-node-compose
-func ConnectToTemplateNodes() ([]Chainlink, error) {
+func ConnectToTemplateNodes() ([]Chainlink, []common.Address, error) {
```

Comment:
This is a bad shortcut to me which'll be removed in the future when we implement k8s deployments. It should be taken out of this client.